python/kserve/kserve/model_server.py

```diff
@@ -80,7 +77,8 @@ class ModelServer:
         enable_grpc (bool): Whether to turn on grpc server. Default: ``True``
         enable_docs_url (bool): Whether to turn on ``/docs`` Swagger UI. Default: ``False``.
         enable_latency_logging (bool): Whether to log latency metric. Default: ``True``.
-        log_config_file (dict): File path containing UvicornServer's log config. Default: ``None``.
+        configure_logging (bool): Whether to configure KServe and Uvicorn logging. Default: ``True``.
```

Comment:
@elukey Can you help review this change as you originally added the `log_config_file`? We are trying to align with Uvicorn here.

Reply:
@yuzisun the change looks good to me, I'll test it in my settings to see if it all works but on paper I am +1